src/test/java/fr/inria/coming/spoon/repairability/repairtools/JKaliTest.java

```diff
@@ -7,7 +7,7 @@
 
 public class JKaliTest {
     @Test
-    public void testJKali1() throws Exception {
+    public void JKaliTest() throws Exception {
         FinalResult result = TestUtills.runRepairability("JKaliTest", "/repairability_test_files/JKali/");
```

Comment:
This needs to be changed to `TestUtills.runRepairability("JKali", "/repairability_test_files/JKali/");`
The first argument is the name of the tool.

Reply:
thanks!